app/assets/scripts/components/common/layers/layer-detection.js

```diff
@@ -1,26 +1,22 @@
 import config from '../../../config';
 
-// Sample ML Data
-// import * as sampleData from './ship-2020-03-11T1647.json';
-import * as sampleData from './ship-2020-03-11T1834.json';
-
 export default {
-  id: 'sample-ml',
-  name: 'Sample ML',
+  id: 'detection',
```

Comment:
I'd recommend changing the id to something more meaningful.
I see you defined it as `shipDetection` when adding it to the layer list. Could that be it?

Reply:
i decided to make it generic (that import should also be generic), because the detection data is for different types (ships, 'multiple', ...etc). I was thinking it was more succinct to use a general detection layer structure and modify based on intended type. what do you think?

Reply:
Maybe we should have different layers for different detections.
I can imagine different spotlights having different (or several) detections. Each detection may warrant a different name, description, legend, etc. no?